test/Conversion/amd/decompose-unsupported-conversions-cdna.mlir

```diff
@@ -1,33 +0,0 @@
-// RUN: triton-opt %s --split-input-file --decompose-unsupported-amd-conversions=arch=gfx942 | FileCheck %s
```

Comment:
Removed these tests, because these cases are covered by LDS optimize pass, which has it's own tests.